scripts/eurostat/regional_statistics_by_nuts/education_enrollment/education_enrollment_preprocess_gen_tmcf.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import io
+import csv
+
+# data_url = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/trng_lfse_04.tsv.gz"
+
+source_tsv = "./demo_r_d3dens.tsv"
+source_csv_long = "./demo_r_d3dens.csv"
+cleaned_csv = "./PopulationDensity_Eurostat_NUTS3.csv"
+tmcf = "./PopulationDensity_Eurostat_NUTS3.tmcf"
+
+output_columns = [
+    'Date',
+    'GeoId',
+    'Count_Person_PerArea',
+]
+
+
+def translate_wide_to_long(source_tsv):
+    df = pd.read_csv(source_tsv, delimiter='\t')
```

Comment:
I think we need to download the TSV in the code.  This will permit running the code to update the data without any human intervention.

I think you can just give the URL to read.csv: https://stackoverflow.com/questions/32400867/pandas-read-csv-from-url/41880513#41880513 and https://github.com/datacommonsorg/data/blob/master/scripts/us_fed/treasury_constant_maturity_rates/generate_csv_and_mcf.py#L74.

Reply:
I tried it at the time and the issue was that the url data point to *.tsv.gz file and at that time I had difficulty unzipping .gz file.